sql/core/src/main/scala/org/apache/spark/sql/execution/CommandResultExec.scala

```diff
@@ -40,6 +40,8 @@ case class CommandResultExec(
 
   override def innerChildren: Seq[QueryPlan[_]] = Seq(commandPhysicalPlan)
 
+  override def executeToIterator(): Iterator[InternalRow] = unsafeRows.iterator
```

Comment:
good catch! can we follow `ExecutedCommandExec` and put this method right after `def executeCollect`?

Reply:
Modified, thank you for reviewing.